libs/core/src/lib/object-marker/object-marker.component.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    Component,
+    OnInit,
+    ViewEncapsulation,
+    ChangeDetectionStrategy,
+    Input,
+    OnChanges,
+    ElementRef
+} from '@angular/core';
+import { CssClassBuilder, applyCssClass } from '../utils/public_api';
+
+@Component({
+    // tslint:disable-next-line:component-selector
+    selector: '[fd-object-marker]',
+    template: `<ng-content></ng-content>`,
+    styleUrls: ['./object-marker.component.scss'],
+    encapsulation: ViewEncapsulation.None,
+    changeDetection: ChangeDetectionStrategy.OnPush
+})
+export class ObjectMarkerComponent implements OnChanges, OnInit, CssClassBuilder {
+    /** User's custom classes */
+    @Input()
+    class: string;
+    /**
+     * Glyph (icon) of the Object Status.
+     */
+    @Input()
+    glyph: string;
+
+    /** Whether the Object Status is clickable. */
+    @Input()
+    clickable = false;
+
+    /** @hidden */
+    constructor(private _elementRef: ElementRef) {}
+
+    @applyCssClass
+    /** CssClassBuilder interface implementation
+     * function must return single string
+     * function is responsible for order which css classes are applied
+     */
+    buildComponentCssClass(): string[] {
```

Comment:
should by `private`

Reply:
Signature of the method from class CssClassBuilder does not match